environments/sandbox.yml

```diff
@@ -126,3 +126,10 @@ cname:
   - name: "cdnverify.bulkscan"
     ttl: 300
     record: "cdnverify.hmcts-bulkscan-shutter-sbox.azureedge.net"
+  - name: “toffee”
+    ttl: 300
+    record: "sdshmcts-sbox.azurefd.net"
+    shutter: false
+  - name: "afdverify.toffee"
```

Comment:
also need cdnverify records as well

https://github.com/hmcts/azure-public-dns/blob/features/dtspo182/environments/sandbox.yml#L87-L89

should be this I think:

cdnverify.sdshmcts-toffee-shutter-sbox.azureedge.net